pkg/api/registry/registry.proto

```diff
@@ -6,10 +6,23 @@ option go_package = "github.com/networkservicemesh/api/pkg/api/registry";
 import "google/protobuf/empty.proto";
 import "google/protobuf/timestamp.proto";
 
+message PathSegment {
+  string name      = 1;
+  string id        = 2;
+  string token = 3;
+  google.protobuf.Timestamp expires = 4;
+  map<string, string> metrics = 5;
+}
+message Path {
+  uint32 index = 1;
+  repeated PathSegment path_segments = 2;
+}
+
```

Comment:
Why do we have PathSegment and Path when we don't use them?

Reply:
Hello, it's temporary solution. We have next options for this data structure:
1. Use just an array of tokens, because in our OPA policies we check only them
2. Make a similar data structure and place it in SDK, near chain elements that are related to OPA for Registry
3. Leave this data structure in API as is
4. Make Path data structure common for NetworkService and Registry and move it to another place in API repo, because path for registry and path for networkservice are the same

What do you think of it?

Reply:
Sounds good :)